chains/testnet/synternet.json

```diff
@@ -0,0 +1,28 @@
+{
+  "chain_name": "synternet",
+  "api": [
+      {
+          "address": "https://api-testnet.synternet.com", 
+          "provider": "Synternet"
+      }
+  ], 
+  "rpc": [
+      {
+          "address": "https://rpc-testnet.synternet.com:443", 
+          "provider": "Synternet"
+      }
+  ],
+  "sdk_version": "0.47.11",
+  "coin_type": "118",
+  "min_tx_fee": "800",
+  "assets": [{
+      "base": "uamber",
+      "symbol": "AMBER",
+      "exponent": "6",
+      "coingecko_id": "synternet-synt", 
```

Comment:
Is this required or can be omitted or left empty?

Reply:
It can be empty(at least some jsons have it empty). I'll remove it.

Reply:
i think it adds value if we have link to Coingecko, id is correct.

Reply:
Though it is for testnet ,so technically it is wrong since there is no token registered in coingecko for testnet AFAIK :)

Reply:
Yes, that's the reason I asked :)